midp/midp.js

```diff
@@ -861,40 +861,36 @@ Native["com/sun/midp/util/isolate/InterIsolateMutex.getID0.(Ljava/lang/String;)I
 };
 
 Native["com/sun/midp/util/isolate/InterIsolateMutex.lock0.(I)V"] = function(id) {
-    var ctx = $.ctx;
```

Comment:
This could benefit from an explanation of why we aren't using *asyncImpl* like the other async natives.